config/electron.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * Default settings for Electron applications.
+ */
+module.exports = {
+  settings: {
+    'import/core-modules': ['electron'],
```

Comment:
@sindresorhus switched to this model where the core module lists are not baked in, but made a shared config for Electron that can be maintained by the community. 

Another such config can be made for any of Atom, Meteor, etc..

Avoids a constant stream of breaking-change platform-related PRs, let's people solve such problems quickly. I think it's better than baking the sets into the code.